smithy-swift-codegen/src/main/kotlin/software/amazon/smithy/swift/codegen/StructureGenerator.kt

```diff
@@ -143,21 +143,24 @@ class StructureGenerator(
         val hasMembers = membersSortedByName.isNotEmpty()
 
         if (hasMembers) {
```

Comment:
This file contains the codegen changes.  Every file below is just codegen tests with the new formatting.